lib/plugins/create/templates/aws-nodejs-typescript/webpack.config.js

```diff
@@ -1,9 +1,16 @@
 const path = require('path');
-// eslint-disable-next-line import/no-unresolved
 const slsw = require('serverless-webpack');
 
 module.exports = {
   entry: slsw.lib.entries,
+  resolve: {
```

Comment:
Using [default webpack extensions](https://webpack.js.org/configuration/resolve/#resolve-extensions) plus typescript extensions.

Reply:
You could add `.jsx` here too, just to be sure - if someone uses JSX

Reply:
Since we're adding `.tsx` I think it makes sense.